src/features/tagging/getLivePreviewProps.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, it, expect } from 'vitest';
+import { getLivePreviewProps } from './getLivePreviewProps';
+
+describe('getLivePreviewProps', () => {
+  it('returns the expected props with a given entryId, fieldId and locale', () => {
+    const entryId = 'test-entry-id';
+    const fieldId = 'test-field-id';
+    const locale = 'test-locale';
+
+    const result = getLivePreviewProps({
+      entryId,
+      fieldId,
+      locale,
+    });
+    expect(result).toStrictEqual({
+      'data-contentful-field-id': `${entryId}:${fieldId}:${locale}`,
```

Comment:
Just thinking out loud here, what if the user has a ':' inside of the fieldId? Won't this cause issues to extract the variables from the string?
We had a similar issue with the path in Compose before

Reply:
Yeah defo a good point! I didn't want to reintroduce the 👺 or what we had before. Can you think of a more sensible delimiter?

Reply:
I guess another way to do it could be

```
'data-contentful-field-id': fieldId,
'data-contentful-entry-id': entryId,
'data-contentful-locale': locale`,
```

then no need for the delimiting

Reply:
Maybe a delimiter that is less likely to appear within the variable values? Like the pipe symbol (|) or  tilde symbol (~)? Not sure...

Reply:
Ah yes I like your approach 👏

Reply:
Just one note:
That could be resolve to a lot of markup on the website and bloat the HTML response (on server-side-rendering).
I guess we should give some best practices to customers to only render the tags on a preview system so they don't affect the performance of their website?

Reply:
@chrishelgert yeah makes sense!